helix-term/src/ui/editor.rs

```diff
@@ -1213,6 +1216,7 @@ impl EditorView {
                 commands::scroll(cxt, offset, direction);
 
                 cxt.editor.tree.focus = current_view;
+                cxt.editor.ensure_cursor_in_view(current_view);
```

Comment:
Is this change necessary? `commands::scroll` already respects scrolloff

Reply:
Currently the `commands::scroll` does respects scrolloff. However, if the file like this:

```
----------------------------------------------------------------| cursor here
longlonglonglonglonglonglonglonglonglonglonglonglonglonglonglong[]onglonglonglong
longlonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglong
longlonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglong
longlonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglong
short or empty
short or empty
short or empty
short or empty
short or empty
```

after scroll down

```

longlonglonglonglonglonglonglonglonglonglonglonglonglonglonglong[]onglonglonglong
longlonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglong
longlonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglong
longlonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglonglong
short or empty
short or empty
--------------| cursor here
short or empt[]
short or empty
short or empty
```

the cursor will not be visible